### ISSUE TITLE
Decouple --stats and --lists from logging configuration

### ISSUE THREAD

**Issue by robmv** [borgbackup/borg#1588]
I have multiple machines backing up to a central server where all repositories are append only. The server run a daily prune process on each repository. Every backup start with hundreds of lines about:

```
Synchronizing chunks cache...
Archives: 100, w/ cached Idx: 99, w/ outdated Idx: 0, w/o cached Idx: 1.
...
Reading cached archive chunk index for main-2016-08-09 ...
Merging into master chunks index ...
...
Done.
```

This happen because it is needed to enable the INFO logging level in order to get --stats output.

Is it possible to output --stats (and probably --list) directly to stderr without passing inside the logging framework?

if not possible, change "Reading cached ..." and "Merging into master ..." lines to DEBUG level, and leave only at INFO level:

```
Synchronizing chunks cache...
Archives: 100, w/ cached Idx: 99, w/ outdated Idx: 0, w/o cached Idx: 1.
````

**Comment by ThomasWaldmann** [comment]
It was made that way because the sync process can take rather long, so we didn't want users wondering what's happening right now (or if something is happening).

It might be solved already in master branch (1.1 later) because logging is handled a bit differently there.

**Comment by enkore** [comment]
Yes, this is solved in 1.1.x series. --stats will only show stats.
